src/diff/children.js

```diff
@@ -29,7 +29,15 @@ export function diffChildren(parentDom, newParentVNode, oldParentVNode, context,
 
 	let oldChildrenLength = oldChildren.length;
 
-	childDom = oldChildrenLength ? oldChildren[0] && oldChildren[0]._dom : null;
+	if (oldChildrenLength) {
```

Comment:
Removing the length check around the `for` loop saves 2 bytes and all tests keep passing 🎉